cartography/data/indexes.cypher

```diff
@@ -91,7 +91,11 @@ CREATE INDEX ON :GSuiteGroup(email);
 CREATE INDEX ON :GSuiteGroup(id);
 CREATE INDEX ON :GSuiteUser(email);
 CREATE INDEX ON :GSuiteUser(id);
-CREATE INDEX ON :GCPFunction(id)
+CREATE INDEX ON :GCPFunction(id);
+CREATE INDEX ON :GCPSQLInstance(id);
+CREATE INDEX ON :GCPSQLUser(id);
+CREATE INDEX ON :GCPBigtable(id);
+CREATE INDEX ON :GCPFirestore(id);
```

Comment:
We should add indexes for all the new nodes that we introduce. Please add all the nodes from Bigtable and Firestore like `GCPBigtableClusterBackup`, `GCPBigtableCluster`, etc.